### ISSUE TITLE
ARIA SVG roles on Firefox (vega-view)

### ISSUE THREAD

**Issue by jonathanzong** [vega/vega#3258]
On Firefox, the graphics-documet role is not working as expected with Voiceover on Mac. It works correctly on Chrome/Safari. The issue is that because the SVG element doesn't have a non-default role, Firefox is treating it as an image (instead of as a document). I believe the solution is to move the ARIA role from the container div onto the SVG element.

Steps to reproduce:
- Open page with a vega-view visualization e.g. https://vega.github.io/vega-lite/examples/bar.html
- I used VoiceOver on Mac. Step through the visualization using the screen reader.
- On Chrome, you will be able to step through the axes and each individual bar, and the screen reader will read the ARIA descriptions. On Firefox, you will not be able to step through axes/bars, and it will only say "image" instead of picking up the ARIA description.

Solution:
In https://github.com/vega/vega/blob/master/packages/vega-view/src/aria.js, set the ARIA attributes (importantly, `role="graphics-document"`) on the SVG instead of the container.
